Shared/src/main/java/data/CommandManager.java

```diff
@@ -5,20 +5,32 @@
 
 public class CommandManager {
 
+    private static CommandManager instance = null;
+
+    // These strings are usernames
     private Map<String, ArrayList<Command>> _clientCommands;
```

Comment:
We weren't going to use prefixes, following the [style guide](http://google.github.io/styleguide/javaguide.html#s5.1-identifier-names)